app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The variable name `people` is not descriptive enough. It would be better to use a more descriptive name like `person_instances`.